src/backend/joanie/core/serializers/admin.py

```diff
@@ -1094,3 +1094,64 @@ def get_owner_name(self, instance) -> str:
         otherwise fallback to the username
         """
         return instance.owner.get_full_name() or instance.owner.username
+
+
+class AdminEnrollmentLightSerializer(serializers.ModelSerializer):
```

Comment:
Same here with a new module ?

Reply:
I did it for filterset and viewset. But about serializers, by experience, it's a little bit trickier due to circular import issues.

Reply:
ok so in a dedicated PR

Reply:
https://github.com/openfun/joanie/issues/707